WcaOnRails/app/views/relations/relation.html.erb

```diff
@@ -0,0 +1,49 @@
+<% provide(:title, t('.title')) %>
+
+<div class="container text-center">
+  <h3><%= t '.find_relations' %></h3>
+
+  <%= form_tag relation_url, method: :get, class: "form-inline relations-form" do %>
```

Comment:
This feels kind of messy to me. I think you'll find it cleaner to create a virtual model for this form and to use `simple_form_for` instead. That's what I did for merge_people, and I was very pleased with it:

https://github.com/jfly/worldcubeassociation.org/blob/6b8a42146874f699f1970ff9397cff50cbae23eb/WcaOnRails/app/views/admin/merge_people.html.erb#L12-L14.